core/src/main/java/org/openstack4j/openstack/compute/internal/ServicesServiceImpl.java

```diff
@@ -78,4 +79,38 @@ public ExtService disableService(String binary, String host) {
 
 		return put(ExtService.class, uri("/os-services/disable")).entity(ServiceAction.disable(binary, host)).execute();
 	}
-}
\ No newline at end of file
+
+    /**
+     * Forcefully shuts down a compute service.
+     * <p>Author:Jyothi Saroja Daggu</p>
+     * @Title: forceDownService
```

Comment:
`@Title` is not a valid JavaDoc tag, please do not add those. Also, declaring author on a method level makes no sense. As a project, we promote contributors at https://github.com/openstack4j/openstack4j/graphs/contributors.

Reply:
Removed the tags